tests/spec_tests/query_tests/operators_comparison.yml

```diff
@@ -1052,3 +1148,34 @@ tests:
       - {'': { "a": { "$date": { "$numberLong": "10" } }, "c": { "$date": { "$numberLong": "15" } }, "r": null } }
       - {'': { "a": { "$date": { "$numberLong": "10" } }, "b": { "$date": { "$numberLong": "5" } }, "r": null } }
 
+  - description: correctness test for BETWEEN for documents
```

Comment:
not the point of this ticket at all but in looking at this file i noticed (and correct me if there is a reason this should be the case) it looks like we have no "greater than upper bound" test for between strings, ie, the standard _id:  4 document in these collections

Reply:
Good catch. I can drive-by add that here.